src/.nuget/packages.config

```diff
@@ -3,4 +3,5 @@
   <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00040" />
   <package id="dnx-coreclr-win-x86" version="1.0.0-beta5-11580" />
   <package id="Microsoft.DotNet.BuildTools.ApiTools" version="1.0.0-prerelease" />
+  <package id="dnx-mono" version="1.0.0-beta5-11596" />
```

Comment:
Bummer that we have to pull this down even for windows builds.  Did you consider splitting it out?